tests/test_plexosdb_sqlite.py

```diff
@@ -156,27 +159,32 @@ def test_get_collection_id(db):
 
 @pytest.mark.get_functions
 def test_get_object_id(db):
-    gen_01_name = "gen1"
-    gen_id = db.add_object(
-        gen_01_name, ClassEnum.Generator, CollectionEnum.Generators, description="Test Gen"
-    )
-    assert gen_id
-
-    gen_id_get = db.get_object_id(gen_01_name, class_name=ClassEnum.Generator)
-    assert gen_id == gen_id_get
-
-    # Add generator with same name different category
     gen_01_name = "gen1"
     category_name = "PV Gens"
+
     gen_id = db.add_object(
         gen_01_name,
         ClassEnum.Generator,
         CollectionEnum.Generators,
         description="Test Gen",
         category_name=category_name,
     )
+    assert gen_id
+
+    gen_id_get = db.get_object_id(gen_01_name, class_name=ClassEnum.Generator)
+    assert gen_id == gen_id_get
+
+    # Add generator with same name different category
     with pytest.raises(ValueError):
-        _ = db.get_object_id(gen_01_name, class_name=ClassEnum.Generator)
+        gen_01_name = "gen1"
+        gen_id = db.add_object(
+            gen_01_name,
+            ClassEnum.Generator,
+            CollectionEnum.Generators,
+            description="Test Gen",
+        )
```

Comment:
This is what raises the `ValueError`, no? Maybe we can move line 180 before the `with` statement and remove the `db.get_object_id`

Reply:
Done

Reply:
I think I fixed this